### DISCUSSION TITLE
How do I target a different board than the ones offered in the build.py script?

### DISCUSSION THREAD

**Discussion by davidfhorowitz** [Xilinx/finn#890] [Q&A] [answered]
I am trying to use the IP generated from [](https://github.com/Xilinx/finn-examples/tree/main/build/vgg10-radioml ) on a ZCU111 development board. Is there a way to do this? I've tried just switching the board in the Vivado GUI after opening the stitched IP project but then I can't synthesize because lots of the IP is locked. Any advice? 

Thanks,

David

**Comment by fpjentzsch** [comment, marked as answer]
Hi, can you try to build for the ZCU111 from the get-go by selecting it in the build.py file? See https://github.com/Xilinx/finn-examples/issues/65#issuecomment-1759450118
